src/constants/action-types.js

```diff
@@ -51,130 +50,120 @@ import {ACTION_PREFIX} from './default-settings';
  *
  * export default createStore(reducers, {}, applyMiddleware(taskMiddleware))
  */
-const ActionTypes = keyMirror({
+const ActionTypes = {
```

Comment:
this could be achieved by creating a helper that takes the map object and add the prefix to each and every property

```
Object.keys(types).reduce((acc, key) => ({
  [key]: `${ACTION_PREFIX}${types[key]}`
}), {})
```

I mean you. already did the work so it is not a big deal but something to consider